core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -117,24 +117,6 @@ class AdminApiServiceSpec extends ApiServiceSpec {
       }
   }
 
```

Comment:
negative testing for invalid JSON may be good

Reply:
Spray should handle that.